Add security threat submission beta api docs (#16242)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/security-emailthreatsubmission.md

---
title: "emailThreatSubmission resource type"

> description: "Reports suspected spam, malware or phishing emails to Microsoft 365 Defender for Office."

Reviewer: ```suggestion
description: "Reports suspected spam, malware or phishing emails to Microsoft Defender for Office 365."
```

author: "caigen"
ms.localizationpriority: medium
ms.prod: "security"
doc_type: resourcePageType
---

# emailThreatSubmission resource type

Namespace: microsoft.graph.security

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> An abstract type to report suspected spam, malware or phishing emails to Microsoft 365 Defender for Office. You can also submit false positive cases, that should not have been blocked by Microsoft 365 Defender for Office, for example, emails incorrectly categorized as junk or spam.

Reviewer: ```suggestion
An abstract type to report suspected spam, malware or phishing emails to Microsoft Defender for Office 365. You can also submit false positive cases, that should not have been blocked by Microsoft Defender for Office 365, for example, emails incorrectly categorized as junk or spam.
```

|[review](../api/security-emailthreatsubmission-review.md)|None|Review threat submission from end user by administrator.|

## Properties
| Property                     | Type                         | Description                                                                                            |
|:-----------------------------|:-----------------------------|:-------------------------------------------------------------------------------------------------------|
| attackSimulationInfo         | [security.attackSimulationInfo](../resources/security-attacksimulationinfo.md) | If the email is phishing simulation, this field will not be null.|
| internetMessageId            | String                       | It is used to specify the internet message id of the email which is being submitted. This information is present in the email header.                                                       |

> | originalCategory             | submissionCategory           | The original category of the submission. The possible values are: `notJunk`, `spam`, `phishing` and `malware`. |

Author: ```suggestion
| originalCategory             | submissionCategory           | The original category of the submission. The possible values are: `notJunk`, `spam`, `phishing`, `malware` and `unkownFutureValue`. |
```

> | internetMessageId            | String                       | The internet message id of the submitted email.                                                       |
> | originalCategory             | submissionCategory           | The original category of the submission. The possible values are: `notJunk`, `spam`, `phishing` and `malware`. |
> | receivedDateTime             | DateTimeOffset               | The received date time of the submitted email.                                                        | 
> | recipientEmailAddress        | String                       | The email recipient smtp address string.                                                              |
> | sender                       | String                       | The sender of the submitted email.                                                                    | 
> | senderIP                     | String                       | The sender IP of the submitted email.                                                                 |
> | subject                      | String                       | The subject of the submitted email.                                                                   |
> | tenantAllowOrBlockListAction | [security.tenantAllowOrBlockListAction](../resources/security-tenantalloworblocklistaction.md) | Used to auto add urls, attachments and senders of the email threat submission into tenant allow block list. |
> 

Author: ```suggestion
| internetMessageId            | String                       | It is used to specify the internet message id of the email which is being submitted. This information is present in the email header.                                                       |
| originalCategory             | submissionCategory           | The original category of the submission. The possible values are: `notJunk`, `spam`, `phishing` and `malware`. |
| receivedDateTime             | DateTimeOffset               | It is used to specify the date timestamp of when the email was received?                                                        | 
| recipientEmailAddress        | String                       | It is used to specify the email address (in smtp format) of the recipient who received the email which is being submitted?                                                              |
| sender                       | String                       | It is used to specify the email address of the sender.                                                                    | 
| senderIP                     | String                       | It is used to specify the IP address of the sender who send the email which is being submitted.                                                                 |
| subject                      | String                       | It is used to specify the subject of the email being submitted.                                                                   |
| tenantAllowOrBlockListAction | [security.tenantAllowOrBlockListAction](../resources/security-tenantalloworblocklistaction.md) | It is used to automatically add allows for the components such as URL, file, sender which are deemed bad by Microsoft so that similar messages in the future can be allowed. |

```

|[Create emailThreatSubmission](../api/security-emailthreatsubmission-post-emailthreats.md)|[microsoft.graph.security.emailThreatSubmission](../resources/security-emailthreatsubmission.md)|Create a new [emailThreatSubmission](../resources/security-emailthreatsubmission.md) object.|
|[Get emailThreatSubmission](../api/security-emailthreatsubmission-get.md)|[microsoft.graph.security.emailThreatSubmission](../resources/security-emailthreatsubmission.md)|Read the properties and relationships of an [emailThreatSubmission](../resources/security-emailthreatsubmission.md) object.|
|[review](../api/security-emailthreatsubmission-review.md)|None|Review threat submission from end user by administrator.|

## Properties
#### Properties

| Property                     | Type                         | Description                                                                                            |
|:-----------------------------|:-----------------------------|:-------------------------------------------------------------------------------------------------------|

> | attackSimulationInfo         | [security.attackSimulationInfo](../resources/security-attacksimulationinfo.md) | If the email is phishing simulation, the field will not be null.|

Reviewer: ```suggestion
| attackSimulationInfo         | [security.attackSimulationInfo](../resources/security-attacksimulationinfo.md) | If the email is phishing simulation, this field will not be null.|
```

> | internetMessageId            | String                       | The internet message id of the submitted email.                                                        |

Reviewer: ```suggestion
| internetMessageId            | String                       | It is used to specify the internet message id of the email which is being submitted. This information is present in the email header. |
```

> | originalCategory             | submissionCategory           | notJunk, spam, phishing, malware.                                                                      |

Reviewer: ```suggestion
| originalCategory             | submissionCategory           | The category the email is being reported as. The values are "not junk", "spam", "phishing" and "malware".|
```

> | receivedDateTime             | DateTimeOffset               | The received date time of the submitted email.                                                         | 

Reviewer: ```suggestion
| receivedDateTime             | DateTimeOffset               | It is used to specify the date timestamp of when the email was received?| 
```

> | recipientEmailAddress        | String                       | The email recipient smtp address string.                                                               |

Reviewer: ```suggestion
| recipientEmailAddress        | String                       | It is used to specify the email address (in smtp format) of the recipient who received the email which is being submitted?|
```

> | sender                       | String                       | The sender of the submitted email.                                                                     | 

Reviewer: ```suggestion
| sender                       | String                       | It is used to specify the email address of the sender. | 
```

> | senderIP                     | String                       | The sender IP of the submitted email.                                                                  |

Reviewer: ```suggestion
| senderIP                     | String                       | It is used to specify the IP address of the sender who send the email which is being submitted.|
```

> | subject                      | String                       | The subject of the submitted email.                                                                    |

Reviewer: ```suggestion
| subject                      | String                       | It is used to specify the subject of the email being submitted.                                                                    |
```

> | tenantAllowOrBlockListAction | [security.tenantAllowOrBlockListAction](../resources/security-tenantalloworblocklistaction.md) | Used to auto add urls/attachments/senders of the email threat submission into tenant allow block list. |

Reviewer: ```suggestion
| tenantAllowOrBlockListAction | [security.tenantAllowOrBlockListAction](../resources/security-tenantalloworblocklistaction.md) | It is used to automatically add allows for the components such as URL, file, sender which are deemed bad by Microsoft so that similar messages in the future can be allowed.|
```
